.github/workflows/rust-py.yml

```diff
@@ -0,0 +1,99 @@
+name: Rust-py
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - "*"
+  pull_request:
+    branches:
+      - "*"
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  linux:
+    runs-on: ${{ matrix.platform.runner }}
+    strategy:
+      matrix:
+        platform:
+          - runner: ubuntu-latest
+            target: x86_64
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: 3.11
+
+      - name: Install maturin
+        run: |
+          python3 -m pip install maturin twine
+
+      - name: Build the package
+        run: |
+          maturin build -i python3.11 --release --out dist --target "x86_64-unknown-linux-gnu"
+
+      - name: Test package
+        env:
+          DATAROOM_API_KEY: ${{ secrets.DATAROOM_API_KEY }}
+          DATAROOM_TEST_SOURCE: ${{ secrets.DATAROOM_TEST_SOURCE }}
+          DATAROOM_API_URL: ${{ secrets.DATAROOM_API_URL }}
+
+        # needs to be replaced with the live version of the package / evolving version number
+        run: |
+          python3 -m pip install dist/datago-2025.2.1-cp311-cp311-linux_x86_64.whl
```

Comment:
this part is the remaining hack that I can see, need to make the version number dynamic / related to latest tag. I didn´t check pypi update either, but worked before and this GHA comes from Maturin, so hopes are relatively high

Reply:
done